source/marsGameSound.js

```diff
@@ -20,15 +20,15 @@ this.initialize = function() {
     for ( var i = 0; i < soundSet.length; ++i ) {
         this.loadSound( soundSet[i] );
     }
-    this.future( 0 ).setUpSubtitles();
+    this.future( 0.01 ).setUpSubtitles();
```

Comment:
So my takeaway was that I should just always do future( something very small ) to be safe, since I don't have enough knowledge of VWF and its inner workings to make a better judgement call than that.  That said... this isn't my code, so I should shut up and let Spence (or whoever) answer now... :)